docs/CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Changed
 -------
 - Append sample and genome reference information to the summary output
   file in the ``filtering-chemut`` process
+- Sorting of files in ``alignment-star`` process is done via samtools
```

Comment:
How about "Enable sorting of files in ``alignment-star`` process using samtools"? This way we convey the message that sorting can be performed using either tool